tools/prepare-deploy.js

```diff
@@ -37,6 +37,7 @@ var handlers = {
     "ours.js": false,
     "ours.js.map": false,
     "pako.min.js": true,
+    "papaparse.min.js": true,
```

Comment:
This feels to me like a pretty heavy dependency for what should be an easy task. Unless I'm mistaken, it should be possible to obtain the next field from a string using a regular expression like

    /"(?:[^"]+|"")*"|[^",\r\n]*/

As you apparently do auto-conversion manually, what exactly does papaparse give us in addition to tokenizing the string according to this or a similar regular expression? Is the gain worth the hassle of including an extra dependency?